share/shaders/FlowLine.frag

```diff
@@ -8,21 +8,21 @@ in vec3             vertModel;
 uniform sampler1D   colorMapTexture;
 uniform vec3        colorMapRange;
 uniform bool        singleColor;
-uniform vec4        clipPlanes[6];     // clipping planes in model coordinates
+//uniform vec4        clipPlanes[6];     // clipping planes in model coordinates
 
 
 void main(void)
 {
     // First, let's decide if we want to discard this fragment
-    vec4 positionModel = vec4( vertModel, 1.0 );
+    /*vec4 positionModel = vec4( vertModel, 1.0 );
```

Comment:
It's really hard to see that this code is commented out with the old style C comments (/* */). I believe you can use #ifdefs. Check with @stasj. In any event can you add a comment as to why the code is commented out (and not simply deleted)?

Reply:
It was old code, and has already been removed. I'm not sure #ifdef stuff works in the shader.